Assets/Scripts/Augment/AugmentImplementations/KnockbackEffect.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class KnockbackEffect : MonoBehaviour
+{
+    public void KnockAwayTargets(float pushPower, float radius, Vector3 origin)
+    {
+        Collider[] colliders = Physics.OverlapSphere(transform.position, radius);
+
+        foreach (Collider collider in colliders)
+        {
+            if (collider.gameObject.TryGetComponent<PlayerManager>(out PlayerManager manager))
+            {
+                var rigidbody = manager.gameObject.GetComponent<Rigidbody>();
+                rigidbody.AddExplosionForce(pushPower, origin, radius, 0f, ForceMode.Impulse);
+            }
+        }
+    }
+
+    public void KnockAwayTargetsDirectional(float pushPower, Vector3 normal, PlayerManager source, float radius)
```

Comment:
These methods could be static and have a transform passed in. No need to instantiate another object for this... I think?

Reply:
Removed the file, and used gunController's forward vector instead.